compute/admin_guide/install/system_requirements.adoc

```diff
@@ -185,6 +185,21 @@ Runtime prevent capability is supported only for DNS events. Other prevent capab
 
 SLES 15  SP1 - SP4 - Only host Defenders are supported.
 
+|Talos OS
+|Talos OS 1.3.0
+Kernel version: 5.15.83-talos
+Kubernetes version: 1.25.4
+containerd version: 1.6.12
+
+
+The following features are not available for Talos OS.
```

Comment:
@Pubs-MV do we want to set this as a note? similar to bottlerocket

Reply:
done, and also clarifying that we do not support both Vuln and Compliance block policies